components/automate-ui/src/app/pages/project/rules/project-rules.component.scss

```
@@ -29,6 +29,71 @@ chef-page {
     max-width: 340px;
   }
 
+  /* Custom dropdown */
+  .custom-dropdown {
+    position: relative;
+    vertical-align: middle;
+  }
+
+  .custom-dropdown select {
+    margin: 0;
+    background-color: $chef-white;
+    color: $chef-primary-dark;
+    font-size: inherit;
+    padding: 12px;
+    padding-right: 16px;
+    height: 45px;
+    border-radius: 4px;
+    border: solid 1px $chef-grey;
+    text-indent: 0.01px;
+    text-overflow: '';
+    font-weight: 400;
+    font-stretch: normal;
+    font-style: normal;
+    line-height: 1.5;
+    letter-spacing: 0.2px;
+    outline: none;
+    /*Hiding the select arrow for firefox*/
```

For each of your added comments like this please put use a space on either end, so:
```suggestion
    /* Hiding the select arrow for firefox */
```